### ISSUE TITLE
Some options in backend_config.ini unused?

### ISSUE THREAD

**Issue by aabdnn** [zonemaster/zonemaster-backend#273]
I have been auditing the options in backend_config.ini, and my reading of the code suggests that the following options, even if configured, are not used, because there is no code to actually read them:

```
[LOG]
log_dir=logs/

[PERL]
interpreter=perl
```

Is this true? If so, can these please be removed? They only serve to confuse a user.

**Comment by matsduf** [comment]
Both are used by script/crontab_job_runner/execute_tests.pl

```
my $LOG_DIR = Zonemaster::WebBackend::Config->LogDir();
my $perl_command = Zonemaster::WebBackend::Config->PerlIntereter();
```

**Comment by aabdnn** [comment]
But "execute_tests.pl" is neither documented nor installed. If it's meant to be an integral part of the backend, then please document and install it. If it's just a debug script, then the options it uses should not be part of the sample configuration in backend_config.ini.

**Comment by matsduf** [comment]
Good point.

Options:

* Option 1: Remove the entries from ini and Config.pm
* Option 2: Include execute_tests.pl in installation and document it.

**Comment by mtoma** [comment]
They must be removed as well as the execute_tests.pl script

**Comment by matsduf** [comment]
@mtoma, we do need various helper scripts for backend.

**Comment by matsduf** [comment]
script/crontab_job_runner/execute_tests.pl has now been removed (#642) and now we can also remove "[LOG]" and "[PERL]" from the ini file and Config.pm.

**Comment by matsduf** [comment]
Resolved by #688.
